{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"resourceNotFoundException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","identifier":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/ResourceNotFoundException","preciseIdentifier":"s:15AWSAuditManager25ResourceNotFoundExceptionV","text":"ResourceNotFoundException"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsauditmanager\/updateassessmentframeworkoutputerror\/resourcenotfoundexception(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/UpdateAssessmentFrameworkOutputError\/resourceNotFoundException(_:)","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"resourceNotFoundException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ResourceNotFoundException","preciseIdentifier":"s:15AWSAuditManager25ResourceNotFoundExceptionV"},{"kind":"text","text":")"}],"title":"UpdateAssessmentFrameworkOutputError.resourceNotFoundException(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:15AWSAuditManager36UpdateAssessmentFrameworkOutputErrorO25resourceNotFoundExceptionyAcA08ResourceijK0VcACmF","modules":[{"name":"AWSAuditManager"}]},"hierarchy":{"paths":[["doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager","doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/UpdateAssessmentFrameworkOutputError"]]},"references":{"doc://AWSAuditManager/documentation/AWSAuditManager":{"role":"collection","title":"AWSAuditManager","abstract":[],"identifier":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager","kind":"symbol","type":"topic","url":"\/documentation\/awsauditmanager"},"doc://AWSAuditManager/documentation/AWSAuditManager/UpdateAssessmentFrameworkOutputError/resourceNotFoundException(_:)":{"role":"symbol","title":"UpdateAssessmentFrameworkOutputError.resourceNotFoundException(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"resourceNotFoundException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ResourceNotFoundException","preciseIdentifier":"s:15AWSAuditManager25ResourceNotFoundExceptionV"},{"kind":"text","text":")"}],"abstract":[],"identifier":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/UpdateAssessmentFrameworkOutputError\/resourceNotFoundException(_:)","kind":"symbol","type":"topic","url":"\/documentation\/awsauditmanager\/updateassessmentframeworkoutputerror\/resourcenotfoundexception(_:)"},"doc://AWSAuditManager/documentation/AWSAuditManager/UpdateAssessmentFrameworkOutputError":{"role":"symbol","title":"UpdateAssessmentFrameworkOutputError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"UpdateAssessmentFrameworkOutputError"}],"abstract":[],"identifier":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/UpdateAssessmentFrameworkOutputError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"UpdateAssessmentFrameworkOutputError"}],"url":"\/documentation\/awsauditmanager\/updateassessmentframeworkoutputerror"},"doc://AWSAuditManager/documentation/AWSAuditManager/ResourceNotFoundException":{"role":"symbol","title":"ResourceNotFoundException","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ResourceNotFoundException"}],"abstract":[{"type":"text","text":"The resource that’s specified in the request can’t be found."}],"identifier":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/ResourceNotFoundException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ResourceNotFoundException"}],"url":"\/documentation\/awsauditmanager\/resourcenotfoundexception"}}}